python_modules/dagster/dagster/_core/instance/config.py

```diff
@@ -116,6 +116,20 @@ def dagster_instance_config(
                 dagster_config_dict["storage"],
             )
 
+    # validate default op concurrency limits
+    if "concurrency" in dagster_config_dict:
+        default_concurrency_limit = dagster_config_dict["concurrency"].get(
+            "default_op_concurrency_limit"
+        )
+        if default_concurrency_limit is not None:
+            if default_concurrency_limit < 0 or default_concurrency_limit > 1000:
+                raise DagsterInvalidConfigError(
+                    f"Found value `{default_concurrency_limit}` for `default_op_concurrency_limit`, "
+                    "Expected value between 0-1000.",
+                    [],
+                    None,
+                )
```

Comment:
I think its nice to use an env var overridable const value for this upper bound. I think allowing open source deployments to manipulate their limits if desired is nice for a relatively small cost.

Reply:
yeah, might split that out as a follow-on diff, since there's a couple other places where the max is hard-coded